src/App/App.js

```diff
@@ -1,14 +1,9 @@
-
-import HomePage from '../Pages/HomePage/HomePage';
-
-function App  () {
-  console.log("App Called.");
-  return (
-    <div>
-      <HomePage />
-    </div>
-  );
- 
-};
+import React from 'react';
+import Footer from '../Components/footer/Footer';
+function App() {
+return (
+	<Footer />
```

Comment:
1) Always wrap return statement in a parent. Like <div> or <Fragment>
2) We only import app pages and so routings among pages in App.Js We never import direct components in here